test/oats/kafka/yaml/oats_go_kafka.yaml

```diff
@@ -0,0 +1,44 @@
+docker-compose:
+  generator: generic
+  files:
+    - ../docker-compose-beyla-go-kafka.yml
+input:
+  - path: '/?data'
+
+interval: 2s
```

Comment:
during my initial tests, i changed this from 500ms to 2s, maybe we can change it back?

Reply:
Done.